### PR TITLE
WIP : add tags to TwoWindingsTransformer and Line Infos Mapper

### PR DESCRIPTION
**Pull request by basseche**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java [hunk 1]
```diff
@@ -127,13 +128,22 @@ private static BranchTabInfos toTabInfos(@NonNull final Branch<?> branch, @Nulla
         return toTabBuilder((BranchTabInfosBuilder<BranchTabInfos, ?>) BranchTabInfos.builder(), branch, dcPowerFactor, loadOperationalLimitGroups).build();
     }
 
+    private static List<LimitsProperty> getLimitsProperties(OperationalLimitsGroup group) {
+        ArrayList<LimitsProperty> properties = new ArrayList<>();
+        for (String name : group.getPropertyNames()) {
```
Comment (EtienneLt) [inline src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java]: propertyName instead of name ?

File: src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java [hunk 2]
```diff
@@ -256,7 +277,6 @@ private static List<TemporaryLimitData> toMapDataTemporaryLimit(@NonNull final C
                 .name(l.getName())
                 .acceptableDuration(l.getAcceptableDuration() == Integer.MAX_VALUE ? null : l.getAcceptableDuration())
                 .value(l.getValue() == Double.MAX_VALUE ? null : l.getValue())
-                .build())
-            .collect(Collectors.toList());
+                .build()).toList();
```
Comment (EtienneLt) [inline src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java]: be careful with that, sonar recommends it but it changes the behavior

Comment (EtienneLt) [reply src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java]: Image

**Review by EtienneLt — COMMENTED** [review]
Can you add a test with properties in it (tests are with empty list of properties)

File: src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java (hunk 1 above)
Comment (flomillot) [reply src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java]: also, you can't make it a Stream ?

File: src/test/resources/partial-lines-form-data.json [hunk 3]
```diff
@@ -63,7 +64,8 @@
             "acceptableDuration": 100
           }
         ],
-        "applicability": "SIDE1"
+        "applicability": "SIDE1",
+        "limitsProperties": []
```
Comment (flomillot) [inline src/test/resources/partial-lines-form-data.json]: maybe you should test at least one which is not empty

File: src/test/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapperTest.java [hunk 4]
```diff
@@ -73,15 +74,15 @@ private CurrentLimitsData testResult(final String id, @Nullable final Double per
             }
             final CurrentLimitsData result = BranchInfosMapper.operationalLimitsGroupToMapDataCurrentLimits(olgMock, null);
             if (id != null) {
-                Mockito.verify(olgMock, Mockito.times(permanentLimit == null ? 1 : 2)).getCurrentLimits();
```
[the remark this answers is not in the document]
Comment (basseche) [reply src/test/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapperTest.java]: since we've changed the function we call only once now

**Review by flomillot — APPROVED** [review]
